### PR TITLE
doc/rados/operations/erasure-code.rst: allow recovery below min_size

### PR DESCRIPTION
**Pull request by neha-ojha**
https://github.com/ceph/ceph/pull/17619 allows recovery below min_size
for EC pools starting in Octopus. Also mention that this wasn't allowed
earlier.

Fixes: https://tracker.ceph.com/issues/40488
Signed-off-by: Neha Ojha <nojha@redhat.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by neha-ojha** [comment]
@hoerup can you take a look?

**Comment by gregsfortytwo** [comment]
I updated the text a bit, check if that looks good?

File: doc/rados/operations/erasure-code.rst [hunk 1]
```diff
@@ -165,11 +165,17 @@ More information can be found in the `cache tiering
...
-that as long as there are enough shards available for data to be recovered,
-we will allow recovery to proceed below min_size.
+If an erasure coded pool loses some shards, it must recover them from the others.
+This generally involves reading from the remaining shards, reconstructing the data, and
+writing it to the new peer.
+In Octopus, erasure coded pools can recover as long as there are at least *K* shards
+available. (With fewer than *K* shards, you have actually lost data!)
+
+Prior to Octopus, erasure coded pools required at least *min_size* shards to be
+available. (We generally recommend min_size be *K+2* or more to prevent loss of writes and data.)
```
Comment (tchaikov) [inline doc/rados/operations/erasure-code.rst]: how about:

> Prior to Octopus, erasure coded pools required at least *min_size* shards to be available, even if *min_size* is greater than *K*.

**Comment by hoerup** [comment]
I think @gregsfortytwo text updated @tchaikov addition would solve the documentation need i saw

**Comment by neha-ojha** [comment]
@gregsfortytwo can you make the recommended changes add your signed-off-by?

**Comment by gregsfortytwo** [comment]
Done. FYI all you can edit doc PRs and add commits directly on Github using the menu below "Review Changes" in the "Files Changed" tab, which is how I made these edits.
(I think you can edit anything, actually, but it's easiest with docs.)

**Comment by gregsfortytwo** [comment]
...of course, if you do that and don't add the Signed-off-by: statement, you appear to be stuck since I can't find a way to force-push a PR. :(
Can you edit them to fix that, @neha-ojha?

**Comment by neha-ojha** [comment]
> ...of course, if you do that and don't add the Signed-off-by: statement, you appear to be stuck since I can't find a way to force-push a PR. :(
> Can you edit them to fix that, @neha-ojha?

done

**Comment by neha-ojha** [comment]
jenkins render docs

**Pull request merged**